src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -2603,6 +2604,10 @@ private function addRateLimiterSection(ArrayNodeDefinition $rootNode, callable $
                                             ->integerNode('amount')->info('Amount of tokens to add each interval.')->defaultValue(1)->end()
                                         ->end()
                                     ->end()
+                                    ->enumNode('calendar_interval')
+                                        ->info('Configures the chosen calendar interval if "policy" is set to "calendar_window".')
+                                        ->enumFqcn(CalendarInterval::class)
```

Comment:
wouldn't this break when using FrameworkBundle without the RateLimiter component (and so with CalendarInterval not being defined) ?